### ISSUE TITLE
graph: Suggest an option not showing a first field like '-f none'

### ISSUE THREAD

**Issue by Taeung** [namhyung/uftrace#185]
Currently we can use a `-f none` option when using `uftrace-replay`,
I think it is useful when comparing two call-graph

So I suggest an additional option hiding a first field of `uftrace-graph`.
What do you think about it ? @namhyung , @honggyukim 

#### When a time duration field exists:
```
$ uftrace graph

calling functions
=====================================
   2.077  s : (1) main
 938.867 ms :  +-(1) bench_vector_push_back
 392.761 ms :  |  +-(999980) std::__cxx11::basic_string::_M_construct
 224.755 ms :  |  | (999980) memcpy
            :  |  | 
 470.191 ms :  |  +-(21) std::vector::_M_insert_aux
  91.690 us :  |     +-(21) operator new
            :  |     | 
 353.752 ms :  |     +-(1048596) std::__cxx11::basic_string::_M_construct
 166.511 ms :  |     | (1048596) memcpy
            :  |     | 
   1.627 ms :  |     +-(20) operator delete
            :  | 
 372.736 ms :  +-(1) bench_deque_push_back
 239.278 ms :  |  +-(937501) std::__cxx11::basic_string::_M_construct
  72.919 ms :  |  | (937501) memcpy
            :  |  | 
  52.383 ms :  |  +-(62500) std::deque::_M_push_back_aux
  18.975 ms :  |     +-(62514) operator new
            :  |     | 
  15.956 ms :  |     +-(62500) std::__cxx11::basic_string::_M_construct
   5.010 ms :  |     | (62500) memcpy
            :  |     | 
 452.351 us :  |     +-(14) memmove
            :  |     | 
   9.803 us :  |     +-(14) operator delete
            :  | 
 765.847 ms :  +-(1) bench_list_push_back
 256.058 ms :     +-(1000001) std::__cxx11::basic_string::_M_construct
  78.531 ms :     | (1000001) memcpy
            :     | 
 124.158 ms :     +-(1000000) operator new
            :     | 
  62.868 ms :     +-(1000000) std::__detail::_List_node_base::_M_hook
```

#### No time duration field case:

```
calling functions
=====================================
 (1) main
  +-(1) bench_vector_push_back
  |  +-(999980) std::__cxx11::basic_string::_M_construct
  |  | (999980) memcpy
  |  | 
  |  +-(21) std::vector::_M_insert_aux
  |     +-(21) operator new
  |     | 
  |     +-(1048596) std::__cxx11::basic_string::_M_construct
  |     | (1048596) memcpy
  |     | 
  |     +-(20) operator delete
  | 
  +-(1) bench_deque_push_back
  |  +-(937501) std::__cxx11::basic_string::_M_construct
  |  | (937501) memcpy
  |  | 
  |  +-(62500) std::deque::_M_push_back_aux
  |     +-(62514) operator new
  |     | 
  |     +-(62500) std::__cxx11::basic_string::_M_construct
  |     | (62500) memcpy
  |     | 
  |     +-(14) memmove
  |     | 
  |     +-(14) operator delete
  | 
  +-(1) bench_list_push_back
     +-(1000001) std::__cxx11::basic_string::_M_construct
     | (1000001) memcpy
     | 
     +-(1000000) operator new
     | 
     +-(1000000) std::__detail::_List_node_base::_M_hook
```

**Comment by namhyung** [comment]
I'm ok with it.

**Comment by honggyukim** [comment]
I also like it!

**Comment by honggyukim** [comment]
Hi @Taeung, please take a look at #290.

**Comment by namhyung** [comment]
Merged b7794b0
